.github/workflows/ci.yml

```diff
@@ -0,0 +1,85 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  checks:
+    name: Project Checks
+    runs-on: ubuntu-18.04
+    timeout-minutes: 5
+
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.16.2
```

Comment:
Do you think for these "library" repositories `1.16.x` would perhaps be specific enough? (that way we wouldn't have to update these for patch releases)

Reply:
yup! good suggestion--saves us a lot of tracking effort